remote_vector_index_builder/core/Dockerfile

```diff
@@ -1,12 +1,22 @@
 FROM opensearchstaging/remote-vector-index-builder:faiss-base-latest
 
-WORKDIR /remote_vector_index_builder
+USER root
 
-COPY ./remote_vector_index_builder/core/requirements.txt /remote_vector_index_builder/core/requirements.txt
+WORKDIR /app
 
-RUN pip install --no-cache-dir --upgrade -r /remote_vector_index_builder/core/requirements.txt
+COPY ./remote_vector_index_builder/core/requirements.txt /app/remote_vector_index_builder/core/requirements.txt
 
-COPY ./remote_vector_index_builder/core /remote_vector_index_builder/core
+RUN pip install --no-cache-dir --upgrade -r /app/remote_vector_index_builder/core/requirements.txt
 
-ENV PYTHONPATH='${PYTHONPATH}:/tmp/faiss/build/faiss/python:/remote_vector_index_builder'
-RUN ["python", "core/test_imports.py"]
\ No newline at end of file
+COPY ./remote_vector_index_builder/core /app/remote_vector_index_builder/core
+
+RUN pip install -e ./remote_vector_index_builder/core
+
+COPY ./remote_vector_index_builder/__init__.py /app/remote_vector_index_builder/__init__.py
+
+ENV PYTHONPATH="${PYTHONPATH}:/app"
```

Comment:
do we still need this? if we are doing installs?

Reply:
Only needed for the test_imports
Python gave me an error stating no module named remote_vector_index_builder

And :
We installed the packages core.* and app.*
not the path remote_vector_index_builder.*